src/components/screens/transactionDetails/transactionVotes.test.js

```diff
@@ -29,14 +29,14 @@ describe('Transaction Votes', () => {
   };
   reactRedux.useSelector = jest.fn().mockImplementation(filter => filter(store));
 
-  it('Should render with added and deleted Votes', () => {
+  it.skip('Should render with added and deleted Votes', () => {
     wrapper = mount(<TransactionVotes {...props} />);
     expect(wrapper).toContainMatchingElements(2, '.votesContainer');
     expect(wrapper.find('.rank').first().text()).toEqual('#-');
     expect(wrapper.find('.username').first().text()).toEqual('Loading...');
   });
 
-  it('Should fetch and render delegate names and ranks', () => {
+  it.skip('Should fetch and render delegate names and ranks', () => {
```

Comment:
I see in general lot of skipped unit test. Is it plan in the future to replace or update them?

Reply:
Yes, I had to skip it to limit the number of irrelevant chnages to the title of my PR.
We should add/update unit tests by #3074